### PR TITLE
Fix/cache removal notification xcode7

### PR DESCRIPTION
**Pull request by adamkaplan**

### CODE REVIEW

**Comment by fandyfyf** [comment]
It's good stuff. lgtm!

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: YMCache/YMMemoryCache.h [hunk 1]
```diff
@@ -48,11 +48,11 @@ typedef BOOL(^YMMemoryCacheEvictionDecider)(id key, id value, void *__nullable c
...
  * the cache will post a notification named `kYFCacheItemsChangedNotificationKey` which includes as
- * user info, a dictionary containing all key-value pairs which have been added to the cache since
- * the previous notification was posted.
+ * user info, a dictionary containing all key-value pairs which have been added to, updated in or removed
+ * from the cache since the previous notification was posted.
  *
- * Values which have been removed from the cache prior to the notification are not included in the
- * notification dictionary.
+ * Values which have been removed from the cache prior to the notification will be set to the marker
+ * singelton `[NSNull null]`.
```
Comment (dbgrandi) [inline YMCache/YMMemoryCache.h]: `singleton`

File: YMCacheTests/YMCachePersistenceControllerSpec.m [hunk 2]
```diff
@@ -0,0 +1,157 @@
+//
+//  YMCachePersistenceControllerSpec.m
+//  YMCache
+//
```
Comment (dbgrandi) [inline YMCacheTests/YMCachePersistenceControllerSpec.m]: Can you delete these top boilerplate lines?

File: YMCache/YMMemoryCache.h (hunk 1 above)
Comment (adamkaplan) [inline YMCache/YMMemoryCache.h]: :ghost:

File: YMCacheTests/YMCachePersistenceControllerSpec.m (hunk 2 above)
Comment (adamkaplan) [inline YMCacheTests/YMCachePersistenceControllerSpec.m]: :hammer:

**Comment by adamkaplan** [comment]
Will re-open in fresh PR.

**Comment by adamkaplan** [comment]
Changed my mind ;-)

**Pull request merged**